miden-lib/asm/kernels/transaction/lib/prologue.masm

```diff
@@ -352,7 +352,7 @@ proc.validate_new_account
 
         if.true
             # assert the fungible faucet reserved slot is initialized correctly (EMPTY_WORD)
-            or or or assertz.err=ERR_PROLOGUE_NEW_FUNGIBLE_FAUCET_NON_EMPTY_RESERVED_SLOT
```

Comment:
These previous `or or or` chain would cause this to fail with an error related to the inputs not being binary if any of the elements were not `0`. 
I've changed this to `add add add` in order for this branch to return the expected `ERR_PROLOGUE_NEW_FUNGIBLE_FAUCET_NON_EMPTY_RESERVED_SLOT` error, because we are checking against `EMPTY_WORD` anyway, and `add` takes the same cycles as `or`. But in reality this is probably not really equivalent because of potential overflows?

Reply:
Yeah, we can't use `add add add` because some elements can cancel out. One other option that @phklive used was `padw eqw` but it takes 19 cycles. Maybe we should have something like `is_empty_word` procedure which could look like so:
```
eq.0 swap eq.0 and swap eq.0 and swap eq.0 and
```
This would still take 10 cycles - which is better than 19. Maybe this should be added to `miden-stdlib`.

Reply:
Leaving the `or or or` as it was would work as well for ~one third of the cycles, but the error would not be very indicative. So, not sure what would be preferred.